app/models/ExportsDeclaration.scala

```diff
@@ -216,4 +213,10 @@ case class ExportsDeclaration(
 
 object ExportsDeclaration {
   implicit val format: OFormat[ExportsDeclaration] = Json.format[ExportsDeclaration]
+
+  def isCodePrefixedWith(maybeCode: Option[String], prefix: Seq[Int]): Boolean = maybeCode match {
+    case Some(code) if code.trim.nonEmpty =>
+      prefix.exists(digits => code.startsWith(digits.toString))
```

Comment:
Nit: `digits` ? A different naming maybe? Maybe `prefix` => `prefixes` and `digits` => `prefix` ?

Reply:
Yeah that sounds a bit better